### PR TITLE
[ADD] ignore file odoo.conf

### PR DESCRIPTION
**Pull request by EnriqueCoDev**

### CODE REVIEW

File: odoo.conf [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+limit_request = None
+limit_time_cpu = 60
+limit_time_real = 120
+limit_time_real_cron = None
+max_cron_threads = 2
+osv_memory_age_limit = 1.0
+osv_memory_count_limit = False
+workers = 25
+##
+# Mail Configure
```
Comment (JayVora-SerpentCS) [inline odoo.conf:55]: SMTP section really matters?
We now use it all via outgoing sever IMHO.
Curious.

**Comment by Yenthe666** [comment]
This really shouldn't be here anyways? :) CC @mart-e

**Comment by EnriqueZav96** [comment]
Sorry, I just realized that I did not configure my IDE correctly

File: odoo.conf (hunk 1 above)
Comment (EnriqueZav96) [reply odoo.conf]: hi :D, by my mistake of not having configured my IDE correctly, I ended up doing pull request to repo.

**Pull request closed without merging**